x-pack/plugins/streams_app/public/components/stream_detail_enriching/index.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { StreamDefinition } from '@kbn/streams-plugin/common';
+import React from 'react';
+
+export function StreamDetailEnriching({
+  definition: _definition,
+  refreshDefinition: _refreshDefinition,
+}: {
+  definition?: StreamDefinition;
+  refreshDefinition: () => void;
+}) {
+  return (
+    <>
+      {i18n.translate('xpack.streams.streamDetailEnriching.streamEnrichingLabel', {
+        defaultMessage: 'Stream enriching',
```

Comment:
enrichment? no idea

Reply:
That's just a placeholder, the actual UI will go there. I replaced it with `TODO` to make it clear.